bfxapi/websockets/bfx_websocket.py

```diff
@@ -187,7 +187,7 @@ class BfxWebsocket(GenericWebsocket):
     - `unsubscribed` (Subscription): A channel has been un-subscribed
     """
 
-    def __init__(self, API_KEY=None, API_SECRET=None, host='wss://api-pub.bitfinex.com/ws/2',
+    def __init__(self, host, API_KEY=None, API_SECRET=None,
```

Comment:
hello vigan, thanks for checking

BfxWebsocket is called only from https://github.com/bitfinexcom/bitfinex-api-py/blob/master/bfxapi/client.py#L24 and in that function ws_host defaults to WS_HOST. I wanted to avoid that duplicate hardcoded URL, but I think you're right and it's safer if I use WS_HOST also there so if our users have some highly customized implementation we do not break anything